pkg/arvo/sys/vane/gall.hoon

```diff
@@ -1575,52 +1549,18 @@
   ~|  [%gall-call-failed duct q.hic]
   =/  =task:able  ((harden task:able) q.hic)
   ::
-  =/  initialised  (mo-abed:mo duct)
+  =/  mo-core  (mo-abed:mo duct)
   ?-    -.task
-      %conf
-    =/  =dock  p.task
-    =/  =ship  p.dock
-    ?.  =(our ship)
-      ~&  [%gall-not-ours ship]
-      [~ gall-payload]
-    ::
-    =>  (mo-boot:initialised q.dock q.task)
-    mo-abet
-  ::
+      %conf  mo-abet:(mo-boot:mo-core dap.task our %home)
```

Comment:
This removes the ability to run apps anywhere other than `%home` -- that seems pretty drastic.